.github/stale.yml

```diff
@@ -0,0 +1,58 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 700  # start with a large number and reduce shortly
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7 # start with a large number and reduce over time
```

Comment:
I would rather set to stale after one year, but set daysUntilClose to 4 weeks or so.

Reply:
I agree re one year, but let's start with longer and then reduce as we know it's working?